{
    "error": "HTTPConnectionPool(host='ac2i.tzo.com', port=80): Max retries exceeded with url: /dctc/ChangeLog (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7effc5baa730>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
}